web/modules/custom/joinup_search/joinup_search.module

```diff
@@ -395,3 +396,25 @@ function joinup_search_views_pre_execute(ViewExecutable $view) {
   // Selecting the entity_created option itself won't affect the query.
   $view->getQuery()->sort('entity_created', 'DESC');
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function joinup_search_form_views_exposed_form_alter(array &$form, FormStateInterface $form_state, string $form_id) {
+  $view = $form_state->getStorage('view');
+  if ($view['view']->id() !== 'search') {
+    return;
+  }
+
+  $user_input = $form_state->getUserInput();
+  if (empty($user_input['keys'])) {
+    unset($form['sort_by']['#options']['relevance']);
+
+    if ($user_input['sort_by'] === 'relevance') {
+      $user_input['sort_by'] = 'last-updated-date';
+    }
+    $form_state->setUserInput($user_input);
+  }
+
+  return $form;
```

Comment:
In a form alter hook the `$form` is passed by reference, so it is not needed to return it.